sch

```diff
@@ -6,7 +6,7 @@ url=$(sort ~/.config/search/search | sed 's/:.*//' | dmenu -i -p "Search Engine"
 search=$(sort ~/.config/search/search_history | dmenu -i -p "Search")
 
 # Echo to history file
-if [ ! "$(grep -q "$search" < ~/.config/search/search_history)" ]; then
+if ! grep -q "$search" < ~/.config/search/search_history; then
```

Comment:
Absolutely! Although I'd remove the redirection symbol, because there's no need for it here; `grep` already reads files as with programs like `cat`.

Honestly though, provided the `search_history` file isn't gigantic, I'd probably opt for a `while` `read` loop, to save needing `grep`, but YMMV.